db/schema.rb

```diff
@@ -302,6 +317,10 @@
     t.uuid "school_id", null: false
     t.uuid "lead_provider_id", null: false
     t.uuid "cohort_id", null: false
+    t.string "status", default: "pending", null: false
+    t.string "reason_for_rejection"
+    t.datetime "accepted_at"
+    t.datetime "rejected_at"
```

Comment:
That's funny. Can you re-generate schema from scratch to make sure these belong here?

Reply:
generates the same thing - not sure what's going on here 🤔

Reply:
Someone forgot to commit their schema change I guess. Alright, if it's repeatable then it can stay imo.

Reply:
@callumacrae - Most likely you run a migration on another branch and those changes are still in your database. Schema is a dump of your current database state, hence the changes. You can either rollback the other migration, reset your whole database or delete those manually